rasa/core/lock_store.py

```diff
@@ -200,6 +200,7 @@ def __init__(
         host: Text = "localhost",
         port: int = 6379,
         db: int = 1,
+        username: Optional[Text] = None,
```

Comment:
How is this property extracted from yaml? Double-checking we don't need to do something else there?

Reply:
I'm guessing that I'll have to add a line for `username` in endpoints.yml and it should work. However I'll do a quick test with docker compose and a testbot.

Once that works out I plan to also add unit tests. I didn't think about integration tests yet but if it doesn't take long -- I'll be sure to add that too.

Reply:
> I'll have to add a line for username in endpoints.yml

Yep, exactly 👍🏻

Reply:
@ancalita added an integration test. It doesn't look like unit test will be possible with our given testing setup